app/src/main/res/values-ar/strings.xml

```diff
@@ -293,7 +293,7 @@
     <string name="multi_apply_rules">تطبيق القواعد على تبديل الأوضاع</string>
     <string name="multi_apply_rules_info">تحميل وتطبيق القواعد عند التبديل بين ملفات التعريف</string>
     <string name="multi_user_title">تفعيل دعم تعدد المستخدمين</string>
-    <string name="multi_user_summary">دعم تعدد المستخدمين للنظام (يعمل فقط في الوضع حيث يتم حظر العناصر المحددة)</string>
```

Comment:
Could you please remove all the language related changes from this PR ? This is managed separately using external service ?

Reply:
OK, but shall I keep the change to `app/src/main/res/values/strings.xml` since this is the "main" string?